internal/cli/apps.go

```diff
@@ -1,32 +1,21 @@
 package cli
 
 import (
-	"fmt"
-
 	"github.com/epinio/epinio/internal/cli/usercmd"
 	"github.com/epinio/epinio/internal/manifest"
 	"github.com/epinio/epinio/pkg/api/core/v1/models"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
-var ()
-
 // CmdApp implements the  command: epinio app
 var CmdApp = &cobra.Command{
 	Use:           "app",
 	Aliases:       []string{"apps"},
 	Short:         "Epinio application features",
 	Long:          `Manage epinio application`,
-	SilenceErrors: true,
-	SilenceUsage:  true,
```

Comment:
I don't understand  much about the SilenceUsage flag, when this is absent, the command will just print the usage and we can get rid of lines 24-29?

Reply:
That's correct, in this case is fine to leave everything out, so the `epinio app` will just print the usage, as it was done in the `RunE`.